### PR TITLE
Change FROM image to Red Hat Universal Base Image (UBI) 8

### PR DESCRIPTION
**Pull request by bradfordcp**
This PR changes the `FROM` Dockerfile lines to `registry.access.redhat.com/ubi8/ubi-minimal:latest`.

Additionally, adjustments have been made given the shift from the Debian base to Red Hat.

### CODE REVIEW

File: operator/Dockerfile [hunk 1]
```diff
@@ -4,9 +4,12 @@
 ##########
 # "builder" compiles and tests the code
 # This stage name is important to the Cloud Platform CI/CD infrastructure, and should be preserved
-FROM golang:1.13-stretch as builder
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as builder
```
Comment (jimdickinson) [inline operator/Dockerfile:7]: no need to change this one, this is just what we use to build

File: operator/Dockerfile [hunk 2]
```diff
@@ -49,11 +52,13 @@ COPY operator ./operator
 ARG VERSION_STAMP=DEV
 
 # Build any code not touched by tests (the generated client)
-RUN MO_VERSION=${VERSION_STAMP} mage operator:buildGo
+RUN MO_VERSION=${VERSION_STAMP} /go/bin/mage operator:buildGo
 
 # Second stage
 # Produce a smaller image than the one used to build the code
-FROM alpine:3.9
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
Comment (jimdickinson) [inline operator/Dockerfile:59]: would strongly advise against using `latest` here

**Comment by bradfordcp** [comment]
Should I stick with the major “8” or one of the points? Ie 8.1
-- 

Christopher Bradford

**Comment by bradfordcp** [comment]
Closing as this has been handled elsewhere

**Pull request closed without merging**